src/modalities/tokenization/tokenizer_wrapper.py

```diff
@@ -22,9 +22,15 @@ def get_token_id(self, token: str) -> int:
 
 class PreTrainedHFTokenizer(TokenizerWrapper):
     def __init__(
-        self, pretrained_model_name_or_path: str, max_length: int, truncation: bool = True, padding: str = "max_length"
+        self,
+        pretrained_model_name_or_path: str,
+        truncation: bool = False,
+        padding: bool = False,
+        max_length: int = None,
```

Comment:
```suggestion
        max_length: Optional[int] = None,
```
and import `Optional` in line 2